internal/provider/resources/work_pool_test.go

```diff
@@ -201,3 +229,59 @@ func getWorkPoolImportStateID(workPoolResourceName string, workspaceResourceName
 		return fmt.Sprintf("%s,%s", workspaceID, workPoolName), nil
 	}
 }
+
+var baseJobTemplateTpl = `
+{
+  "job_configuration": {
```

Comment:
I grabbed this from our docs, as we need a fully-functional payload otherwise the API will throw an error.